twitter/View Controllers/TimelineViewController.m

```diff
@@ -86,7 +99,13 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 - (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath {
     TweetCell *cell = [tableView dequeueReusableCellWithIdentifier:@"TweetIdCell"
                                                       forIndexPath:indexPath];
-    [self.tweetModels[indexPath.row] loadNewCell:cell];
+    if (indexPath.row < self.tweetModels.count) {
+        [self.tweetModels[indexPath.row] loadNewCell:cell];
+        self.tweetModels[indexPath.row].delegate = self;
+        if (indexPath.row == self.tweetModels.count - 1) {
```

Comment:
Can leverage properties and vars to make code more readable generally. Don't bother making the change in this project but the difference is between this and: 

`if (hasTweet(forRow: indexPath.row)`
or
`if (indexPath.row == lastTweetIndex)` 
or 
`if (isLastTweet)`
etc.

None of these are great either I guess.